### DISCUSSION TITLE
Failed to install

### DISCUSSION THREAD

**Discussion by huaweiqihua** [Nourepide/ComfyUI-Allor#22] [Q&A]
Failed to install 'https://github.com/Nourepide/ComfyUI-Allor.git'
When I use ComfyUI for installation, it prompts me that I cannot install this plugin. I see in the prompt that it's necessary to disable the default node override function, but I can't find the related setting in the ComfyUI settings.
<img width="1405" alt="截屏2023-12-18 22 21 58" src="https://github.com/Nourepide/ComfyUI-Allor/assets/77044252/fa4f7ccc-8c4b-4c16-b0b1-72899dca621b">

**Comment by Nourepide** [comment]
Hi.

I've encountered a similar problem in discussions before. Usually it was encountered by those who already had Allor installed before.

Try to go to `custom_nodes` folder and completely uninstall Allor and then install it again.

Please report on the result.

PS:

> I see in the prompt that it's necessary to disable the default node override function

No, this has been [disabled by default](https://github.com/Nourepide/ComfyUI-Allor/issues/4#issuecomment-1695451809) since August.

> but I can't find the related setting in the ComfyUI settings.

If you want you can see [here](https://nourepide.github.io/ComfyUI-Allor-Doc/configuration.html).

**Reply by huaweiqihua** [reply]
Thanks Reply!
I followed the method you provided, deleted the relevant files and reinstalled, but the problem still exists, unfortunately. . . .
I'll think of other ways...

**Reply by Nourepide** [reply]
As I understand it, the manager does not notify about broken imports but finds errors in the ComfyUI output.

When you start ComfyUI do you have a console or terminal? If yes, there should be a detailed description of the error and what happened.

This is the information I will need to help you.

By the way, it is better to know in advance which ComfyUI you are using Windows Portable/Windows/Linux/macOS.
